spec/factories/course_levels.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :course_level, class: 'Course::Level' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.